src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -4415,6 +4422,7 @@ internal static void RebuildConversionCache()
                     CacheConversion<object>(typeofNull, type, LanguagePrimitives.ConvertNullToNumeric, ConversionRank.NullToValue);
                 }
 
+                CacheConversion<string>(typeof(BigInteger), typeofString, ConvertNumericToString, ConversionRank.NumericString);
```

Comment:
This is not needed, because for each element of `s_numericTypes`, the convert-to-string conversion has already been cached in the `foreach` loop right above.

Reply:
```suggestion
```